pkg/resources/tag_masking_policy_association_acceptance_test.go

```diff
@@ -29,6 +31,68 @@ func TestAcc_TagMaskingPolicyAssociation(t *testing.T) {
 	})
 }
 
+func TestAcc_TagMaskingPolicyAssociation1(t *testing.T) {
```

Comment:
nit-pick: `TestAcc_TagMaskingPolicyAssociation1` -> this test deserves a better name than just an ordinal

Reply:
renamed